README.md

```diff
@@ -63,3 +63,27 @@ See how to use plugins in plugin documentation.
 #### C/C++
 
 - [dogebuild-c](https://github.com/dogebuild/dogebuild-c) - a C/C++ build plugin
+
+#### Go lang
+
+- [dogebuild-go](https://github.com/dogebuild/dogebuild-go) - a Go lang build plugin
```

Comment:
Go plugin is still in development phase, so I would prefer to not include it list of plugins.

Reply:
done